docs/REST API/rest-api.md

```diff
@@ -7528,6 +7530,66 @@ Returns information about the specified software version.
 }
 ```
 
+## Vulnerabilities
+
+## Software
```

Comment:
Is the idea here to carve out a new "Vulnerabilities" section that appears in the right-side bar?

If so, I think we want to get rid of the `## Software` title here.

Also, I would consider putting the vulnerabilities endpoints under "Software" because it's consistent with the IA in the UI. That said, I can see the argument for breaking out a new section for the API user.